sass/compat/themes/buddyboss-theme/__imports.scss

```diff
@@ -0,0 +1,16 @@
+// ------------------------------------
+// DEFINE LOCAL VARIABLES
+// ------------------------------------
+$_body-page-selector:                  '.woocommerce-checkout' !default;
+$_body-theme-selector:                 '.theme-buddyboss-theme' !default;
+
+
+
+
+
```

Comment:
Missing the CSS variables for the field height and border radius.

As explained, Those variables affect the height and border of `select2` fields, which most of the time the theme itself has it wrong. But they are also used to calculate the position for the validation check icon and other things. That is why they need to be the same value as from the theme.